CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 
 ### Added
 * New detector `FindBadEndOfStreamCheck` for new bug type `EOS_BAD_END_OF_STREAM_CHECK`. This bug is reported whenever the return value of java.io.FileInputStream.read() or java.io.FileReader.read() is first converted to byte/int and only thereafter checked against -1. (See [SEI CERT rule FIO08-J](https://wiki.sei.cmu.edu/confluence/display/java/FIO08-J.+Distinguish+between+characters+or+bytes+read+from+a+stream+and+-1))
+### Fixed
+- Class names containing the substring "immutable" (case insensitive) are considered immutable
```

Comment:
I can't see any logic based on "class name contains 'immutable'"; is this still accurate?

Reply:
This conversation can be resolved, since it got fixed in https://github.com/spotbugs/spotbugs/pull/1619/commits/65a43889a16ec6fc0f4c919e4cd4710a0e8d4966.